examples/apache/build-apache.acb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env acbuild-script
+
+# Start the build with an empty ACI
+begin
+
+# Name the ACI
+set-name example.com/apache
+
+# Based on alpine
+dep add quay.io/coreos/alpine-sh
+
+# Install apache
+run -- apk update
```

Comment:
ahhhh, saw your comment that said it will automatically add if it's missing, nvm! so this should work regardless.